### ISSUE TITLE
Tables need headers

### ISSUE THREAD

**Issue by sean-fitzpatrick** [APEXCalculus/APEXCalculusPTX#441]
This is just so I remember to do this: for accessibility reasons (screen reader compatibility), the first row in a table typically should have `header="yes"` as an attribute. This tells the reader that the items in the first row are column headers that describe what is in each column.

The only thing I'm unsure of is whether we also do this for cases where we have a `tabular` inside a `figure`. Technically this is not allowed but you'll recall there was a lengthy discussion once with @davidfarmer about how most of our "tables" are not true tables according to the Chicago style. I think David convinced us, but not Rob, since it's still the case that `tabular` isn't allowed anywhere other than in a `table`.

**Comment by sean-fitzpatrick** [comment]
I've tackled this in #444. There are still some tabulars where there isn't a header row. That suggests that perhaps they shouldn't be a tabular...

There is a bigger issue, but I'll open a new issue with a more appropriate title.
